components/widgets/inputs/users_emails_input.jsx

```diff
@@ -188,10 +199,57 @@ export default class UsersEmailsInput extends React.Component {
         IndicatorsContainer: () => null,
     };
 
+    handleInputChange = (inputValue, action) => {
+        if (action.action === 'input-blur') {
+            const values = this.props.value.map((v) => {
+                if (v.id) {
+                    return v;
+                }
+                return {label: v, value: v};
+            });
+
+            for (const option of this.state.options) {
+                if (this.props.inputValue === option.username || this.props.inputValue === ('@' + option.username)) {
+                    this.onChange([...values, option]);
+                    this.props.onInputChange('');
+                    return;
+                } else if (this.props.inputValue === option.email) {
+                    this.onChange([...values, option]);
+                    this.props.onInputChange('');
```

Comment:
Why two conditions it seems that they both execute the same code block in each case? 🤔